extensions/cornerstone/src/init.tsx

```diff
@@ -76,18 +76,13 @@ export default async function init({
     customizationService,
     uiModalService,
     uiNotificationService,
-    cineService,
     cornerstoneViewportService,
     hangingProtocolService,
     toolGroupService,
     viewportGridService,
     stateSyncService,
   } = servicesManager.services as CornerstoneServices;
 
-  window.services = servicesManager.services;
```

Comment:
please don't remove them they are here for debugging the demos

Reply:
I always search for `window.`, `debug` and `console.` when I run `git diff` before pushing and I deleted those lines by accident